qa/abstracts/src/main/java/io/camunda/zeebe/process/test/qa/abstracts/assertions/AbstractProcessInstanceAssertTest.java

```diff
@@ -1435,4 +1439,47 @@ public void testShouldCaptureLatestValueOfVariable()
           .hasVariableWithValue(ProcessPackLoopingServiceTask.TOTAL_LOOPS, "3");
     }
   }
+
+  @Nested
+  class CustomMapperTests {
```

Comment:
🔧 This class is huge 😅 Let's create a new test class to verify the custom mapper.